test/e2e_node/critical_pod_test.go

```diff
@@ -59,7 +59,7 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
 				framework.Skipf("unable to run test without dynamic kubelet config enabled.")
 			}
 			// because adminssion Priority enable, If the priority class is not found, the Pod is rejected.
-			systemCriticalPriority := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: systemCriticalPriorityName}, Value: scheduling.SystemCriticalPriority}
+			systemCriticalPriority := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: systemCriticalPriorityName}, Value: scheduling.HighestUserDefinablePriority}
```

Comment:
From my understanding of critical pods, the priority has to be higher than `scheduling.HighestUserDefinablePriority`.

There should be two priority classes created at cluster bootstrap called `system-node-critical` and `system-cluster-critical` (https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/scheduling/helpers.go#L27). I think we should use one of those in this test.

Reply:
agree, I expected one of those two to be used, along with their priority value